client/src/components/map/types.d.ts

```diff
@@ -11,6 +11,9 @@ export interface CustomMapProps extends MapProps {
    */
   viewState?: Partial<ViewState>;
 
+  /** An string that defines the rotation axis */
+  axis?: string;
```

Comment:
```suggestion
  axis?: 'x' | 'y';
```

Reply:
Also, as we talked, maybe is better to rename this property to something more self-explanatory. `constrainedAxis` 

What about the values. Should we use `x` and `y` or `latitude` and `longitude`